packages/core-web/src/styles/markbind.css

```diff
@@ -120,7 +120,7 @@ code.hljs.inline {
 header[fixed] {
     position: fixed;
     width: 100%;
-    z-index: 1000;
```

Comment:
One more minor styling issue: (navbar example in ug)

![Untitled](https://user-images.githubusercontent.com/3306138/107049700-60050900-6805-11eb-9acd-0be78d006d7a.png)

Reply:
Thanks! Fixed :)